docs/usage/configuration-options.md

```diff
@@ -421,6 +421,37 @@ If enabled, all issues created by Renovate are set as confidential, even in a pu
 !!! note
     This option is applicable to GitLab only.
 
+## configMigration
+
+If enabled, Renovate will raise a pull request if config file migration is needed.
+
+As Renovate evolves and new feature are added, configuration options might evolve as well.
+For such cases, Renovate will raise config migration pull requests.
+
+Example:
+
+When [`baseBranches`](https://docs.renovatebot.com/configuration-options/#basebranches) was introduced, such pull request would offer the following changes:
+
+From
+
+```json
+{
+  "baseBranch": "main"
+}
+```
```

Comment:
We could use a `diff` style display, instead of spreading the info over two sections.

Do we think our readers will understand the `diff` style display?

```diff
{
-  "baseBranch": "main"
+  "baseBranches": ["main"]
}
```

This works on a preview of our docs site:

![diff-display-renovate-docs-preview](https://user-images.githubusercontent.com/34918129/168790897-f37e7ee9-7a14-4dc3-bc23-1518868bc3de.png)

Reply:
looks better to me at least, as the actual change is more obvious and visible.

Reply:
I can't create a suggestion on GitHub with a `diff` in it. Can you please make the change yourself?